app/src/main/java/com/uni/gruppenphaseandroid/playingfield/Figure.java

```diff
@@ -0,0 +1,84 @@
+package com.uni.gruppenphaseandroid.playingfield;
+
+public class Figure {
+    private int id;
+    private Color color;
+
+    public Figure(int id, Color color) {
+        this.id = id;
+        this.color = color;
+    }
+
+    public Field move (Field field, int fieldsToMove) { // Input von Karten: wie viel fahren
+        Field newPosition = field.getFieldAtDistance(fieldsToMove, color);
+            if (newPosition.getCurrentFigure() == null) {
+                newPosition.setCurrentFigure(this);
+            } else {
+                Figure beaten = newPosition.getCurrentFigure(); // figure was beaten and has to be set to Starting Area
+                beaten.getRightStartingArea(field);
+            }
+        return field;
+    }
+
+    public Field getRightStartingArea (Field field) { // if figure is beaten: find an empty space in the right Starting Area
```

Comment:
Vorschlag: Stattdessen die Methode getRightStartingAreaField(Color color) aus PlayingField aufrufen.

Reply:
Okay, danke. Hab ich in deinem Code wohl übersehen.